New addon RAWR

geno3302 opened the pull request:
RAWR plays a sound effect when one of the unity leaders mentions one of
the dragons in chat.

geno3302 committed:
New addon RAWR

RAWR plays a sound effect when one of the unity leaders mentions one of
the dragons in chat.

z16 commented on addons/RAWR/RAWR.lua:
```diff
@@ -0,0 +1,43 @@
+_addon.name = 'RAWR'
+_addon.author = 'Genoxd'
+_addon.version = '1.0.0.0'
+_addon.commands = {'rawr'}
```
You don't need to set a command if the addon does not support any commands.

z16 commented on addons/RAWR/RAWR.lua:
```diff
@@ -0,0 +1,43 @@
+_addon.version = '1.0.0.0'
+_addon.commands = {'rawr'}
+
+require 'lists'
```
You should change this to `require('tables')`, since you don't actually use any lists in here, only the Windower-specific tables.

z16 commented:
1. The addon is missing the license disclaimer
2. You need an entry in the `addons.xml` file for it to show up in the Windower launcher
3. The sound file should be in a different folder, the `data` sub-folder is for user-specific items only

geno3302 committed:
Made requested changes

Removed command.
Changed list to require('tables')
Added license disclaimer.
Added entry to addons.xml
Moved sound file from data to new sounds folder

z16 commented on addons/RAWR/RAWR.lua:
```diff
@@ -0,0 +1,68 @@
+--Copyright (c) 2016, geno3302
```
Please change that to `©`.

z16 commented on addons/RAWR/RAWR.lua:
```diff
@@ -0,0 +1,68 @@
+--    * Redistributions in binary form must reproduce the above copyright
+--      notice, this list of conditions and the following disclaimer in the
+--      documentation and/or other materials provided with the distribution.
+--    * Neither the name of <addon name> nor the
```
Enter your addon name here :)

geno3302 committed:
Made requested changes to copyright

Copyright © and missed addon name
